src/components/AccountListener.js

```diff
@@ -229,34 +230,31 @@ const AccountListener = (props) => {
       })
     }
 
-    await Promise.all([loadBalances(), loadAllowances(), loadRebaseStatus()])
+    const loadLifetimeEarnings = async () => {
+      if (!account) return
+
+      const xusdBalance = AccountStore.currentState.balances['xusd']
```

Comment:
is this the right way to access AccountStore versus `useStoreState(AccountStore, (s) => s.balance.xusd`?

Reply:
Yeah I think so, at least it's done in a bunch of other places - useStoreState is for hooking react updates on state change